libs/galley-types/src/Galley/Types.hs

```diff
@@ -433,16 +443,18 @@ instance ToJSON Conversation where
         , "members" .= cnvMembers c
         , "last_event"      .= ("0.0" :: Text)
         , "last_event_time" .= ("1970-01-01T00:00:00.000Z" :: Text)
+        , "team"    .= cnvTeam c
         ]
 
 instance FromJSON Conversation where
    parseJSON = withObject "conversation" $ \o ->
-       Conversation <$> o .: "id"
-                    <*> o .: "type"
-                    <*> o .: "creator"
-                    <*> o .: "access"
-                    <*> o .: "name"
-                    <*> o .: "members"
+       Conversation <$> o .:  "id"
+                    <*> o .:  "type"
+                    <*> o .:  "creator"
+                    <*> o .:  "access"
+                    <*> o .:? "name"
```

Comment:
Apparently not, but as you can see in the definition of `Conversation` above, `name` is an optional field.